### PR TITLE
web pages responsive styling

### PR DESCRIPTION
**Pull request by vishnusn77**
I'll do the rest pages all together in one pr

### CODE REVIEW

File: Server/Views/components/footer.ejs
Comment (Rahnken) [inline Server/Views/components/footer.ejs:1]: Really not the way we should do this. adding inline styles with important is not a sustainable way to make these , we will adjust this later

File: Server/Views/content/services.ejs
Comment (Rahnken) [inline Server/Views/content/services.ejs:1]: Again inline styles not ideal but will work for this release

**Review by Rahnken — APPROVED** [review]
This is fine for this release, I will go through it when I have a bit of time, some best practices need to be implemented but that is Tech Debt for now

**Comment by vishnusn77** [comment]
Yes, I didn't really have time yesterday. Will definitely implement some best practices later on.

**Pull request merged**